java.hints/src/org/netbeans/modules/java/hints/jdk/ConvertToVarHint.java

```diff
@@ -65,8 +72,10 @@
                     "compiler.err.generic.array.creation" //NOI18N
             )));
 
-    @TriggerPattern("$mods$ $type $var = $init") //NOI18N
-
+    @TriggerPatterns({
+        @TriggerPattern("$mods$ $type $var = $init"), //NOI18N
+        @TriggerPattern("for ($type $var : $expression { $stmts$; }") //NOI18N
```

Comment:
Seems that closing `)` is missing in pattern. It works at runtime, but still worth to correct.